.circleci/config.yml

```diff
@@ -59,6 +73,23 @@ jobs:
         paths:
         - ~/.m2
 
+  # Run Tests
+  tests-java11:
+    working_directory: ~/fabric8io/fabric8-maven-plugin
+    machine: true
```

Comment:
How are we ensuring that we're building on Java 11? I don't see any jdk11 image being used.

Reply:
The -Pjava11 flag, this profile has configured maven-compiler-plugin to use java 11.